framework/python/src/core/session.py

```diff
@@ -658,7 +667,7 @@ def _remove_invalid_questions(self, questions):
         valid_questions.append(question)
 
       else:
-        LOGGER.debug(f'Removed unrecognised question: {question["question"]}')
+        LOGGER.debug(f'Removed unrecognised question: {question["question"]}') # pylint: disable=W1405

```

Comment:
Is the pylint disable necessary? We were never notified of a pylint issue before.

Reply:
I saw this error when running pylint.
![Screenshot from 2024-12-16 12-44-37](https://github.com/user-attachments/assets/5bef0566-2ecd-488d-b998-b4bef054ec4c)